src/App.tsx

```diff
@@ -20,17 +25,133 @@ class App extends React.Component<{}, State> {
       './img/9.png',
       './img/10.png',
     ],
+    step: 3,
+    frameSize: 3,
+    itemWidth: 130,
+    animationDuration: 1000,
+    infinite: false,
+  };
+
+  handleInputChange = (
+    e: React.ChangeEvent<HTMLInputElement>,
+    key: keyof State,
+  ) => {
+    const { value } = e.target;
+
+    this.setState(prevState => ({
+      ...prevState,
+      [key]: +value,
+    }));
+  };
```

Comment:
you can make one handler for all inputs.
something like that.

```suggestion
  handleInputChange = (
    e: React.ChangeEvent<HTMLInputElement>,
  ) => {
    const { value, name, checked } = e.target;

    this.setState(prevState => ({
      ...prevState,
      [name]: name === 'infinite' ? checked : +value,
    }));
  };
```